lambda/rotator-snowflake/setup/setup.go

```diff
@@ -0,0 +1,71 @@
+package setup
+
+import (
+	"database/sql"
+	"os"
+
+	"github.com/chanzuckerberg/go-misc/databricks"
+	"github.com/chanzuckerberg/go-misc/snowflake"
+	"github.com/pkg/errors"
+	"github.com/xinsnake/databricks-sdk-golang/aws"
+)
+
+func Snowflake() (*sql.DB, error) {
+	account, present := os.LookupEnv("SNOWFLAKE_ACCOUNT")
```

Comment:
I sometimes like to use https://github.com/kelseyhightower/envconfig for things like this